### ISSUE TITLE
[DEV] Remove `manual` wiki contents

### ISSUE THREAD

**Issue by rahulporuri** [fossunited/fossunited#669]
### Issue description

Now that https://fossunited.github.io/fossunited/ is up, we can remove

* remove the wiki pages
* remove the `/manual` page from the website
* ~remove the wiki app from `fossunited.org`~

**Comment by rahulporuri** [comment]
This was completed from the frappe app - the wiki pages under `/manual` were deleted from the UI and the `/manual` wiki space was deleted.

there are more wiki pages that need to be migrated to the FOSS United Platform before we can delete the wiki app from the bench. I'll open a separate issue for that migration.

**Comment by rahulporuri** [comment]
I just discovered where the link to the old `/manual` page from the homepage was - under `explore/` - which has now been deleted
